app/org/maproulette/models/dal/TaskDAL.scala

```diff
@@ -362,13 +362,22 @@ class TaskDAL @Inject()(override val db: Database,
 
       task match {
         case Some(t) =>
-          // If the status is changing and if we have a bundle id, then we need
-          // to clear it out along with any other tasks that also have that
-          // bundle id -- essentially breaking up the bundle. Otherwise this
-          // task could end up with a different status than other tasks
-          // in that bundle.
-          if (cachedItem.status != t.status && t.bundleId != None) {
-            this.deleteTaskBundle(user, t.bundleId.get)
+          if (cachedItem.status != t.status) {
+            // If the status is changing and if we have a bundle id, then we need
+            // to clear it out along with any other tasks that also have that
+            // bundle id -- essentially breaking up the bundle. Otherwise this
+            // task could end up with a different status than other tasks
+            // in that bundle.
+            if (t.bundleId != None) {
+              this.deleteTaskBundle(user, t.bundleId.get)
+            }
+
+            // If we are moving this task back to a created status, then we don't need to do a review on it.
+            if (t.status.getOrElse(-1) == Task.STATUS_CREATED) {
```

Comment:
I feel like you are doing this twice. The mergeUpdate function already does this anyway, so I am curious if you actually need to execute this delete again.

Reply:
Removed it.